### PR TITLE
Remove all old panel stuff from collections and revamp views

### PR DESCRIPTION
**Pull request by straleyb**

### CODE REVIEW

**Comment by straleyb** [comment]
Screen Shot 2021-03-01 at 1 03 36 PM
Screen Shot 2021-03-01 at 1 03 44 PM
Screen Shot 2021-03-01 at 1 03 56 PM
Screen Shot 2021-03-01 at 1 04 05 PM

File: app/views/collections/_edit_slides_in_collection.html.erb [hunk 1]
```diff
@@ -1,8 +1,6 @@
-<div class="panel panel-primary">
-  <div class="panel-heading">
-    <h3 class="panel-title">Edit slides in collection</h3>
-  </div>
-  <div class="panel-body">
+<h1>Edit slides in collection</h1>
```
Comment (CGillen) [inline app/views/collections/_edit_slides_in_collection.html.erb:1]: If I'm following this correctly, this goes under the `Editing Collection` H1. There should only be one H1 per page and it should usually be similar to the `<title>` element.
Set this and the main `Edit collection` section to `h2`

File: app/views/collections/_form.html.erb [hunk 2]
```diff
@@ -11,12 +11,12 @@
     </div>
   <% end %>
 
+  <h1>Edit collection</h1>
```
Comment (CGillen) [inline app/views/collections/_form.html.erb:14]: `<h2>` see above

File: app/views/collections/edit.html.erb [hunk 3]
```diff
@@ -1,6 +1 @@
-<h1>Editing Collection</h1>
```
Comment (CGillen) [inline app/views/collections/edit.html.erb:1]: Drop a consistent `<h1>` back at this level. Give it the `sr-only` class if you don't want to display it.

File: app/views/collections/index.html.erb [hunk 4]
```diff
@@ -1,42 +1,30 @@
...
-            <tr>
-              <td><%= link_to collection.name, collection %></td>
-              <td>
-                <%= collection.slides.count.to_s %>
-              </td>
-              <td><%= link_to 'Edit', edit_collection_path(collection) %></td>
-              <td><%= link_to 'Destroy', collection, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-            </tr>
-          <% end %>
+<h3>All collections</h3>
```
Comment (CGillen) [inline app/views/collections/index.html.erb:6]: `<h2>`
Don't skip headers when increasing depth

File: app/views/collections/new.html.erb [hunk 5]
```diff
@@ -5,34 +5,25 @@
...
-      <div class="panel panel-primary">
-        <div class="panel-heading">
-          <h3 class="panel-title">Create new collection</h3>
-        </div>
-        <div class="panel-body">
-          <div class="field">
-            <%= f.label :name %>
-            <%= f.text_field :name, class: "form-control" %>
-          </div>
+      <h3>Create new collection</h3>
```
Comment (CGillen) [inline app/views/collections/new.html.erb:15]: `<h2>` see above

File: app/views/collections/show.html.erb [hunk 6]
```diff
@@ -1,161 +1,40 @@
...
+<hr style="border: 1px solid darkgrey;">
+
+<h2>Slides in collection</h2>
   <!-- Table -->
   <table class="table table-striped">
     <thead>
       <tr>
         <th>Slide</th>
         <th>Title</th>
-        <th>Caption</th>
```
Comment (CGillen) [inline app/views/collections/show.html.erb:15]: Captions provide important information for at least the Donor Coverflow kiosk. I don't know if we should hide it

File: app/views/collections/index.html.erb [hunk 7]
```diff
@@ -1,42 +1,30 @@
...
+  <table class="table table-striped">
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Slides</th>
+        <th colspan="3"></th>
```
Comment (CGillen) [inline app/views/collections/index.html.erb:12]: This pattern should include a header for this column, like `Actions` and the links should be in a `<ul>` in one column

**Review by CGillen — CHANGES_REQUESTED** [review]

**Pull request merged**